diff --git a/dist/index.js b/dist/index.js
index 825b3afa0aa94a98fe188c1605eab8a17f72e844..35ea7ce92c1c6f5d0669fde451809db193ff54f5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -20,13 +20,13 @@ function loader(source) {
     var _a;
     const loaderContext = this;
     // check if plugin is installed
-    if (!errorEmitted &&
-        !loaderContext['thread-loader'] &&
-        !loaderContext[plugin_1.default.NS]) {
-        loaderContext.emitError(new Error(`vue-loader was used without the corresponding plugin. ` +
-            `Make sure to include VueLoaderPlugin in your webpack config.`));
-        errorEmitted = true;
-    }
+    // if (!errorEmitted &&
+    //     !loaderContext['thread-loader'] &&
+    //     !loaderContext[plugin_1.default.NS]) {
+    //     loaderContext.emitError(new Error(`vue-loader was used without the corresponding plugin. ` +
+    //         `Make sure to include VueLoaderPlugin in your webpack config.`));
+    //     errorEmitted = true;
+    // }
     const stringifyRequest = (r) => loaderUtils.stringifyRequest(loaderContext, r);
     const { mode, target, sourceMap, rootContext, resourcePath, resourceQuery: _resourceQuery = '', } = loaderContext;
     const rawQuery = _resourceQuery.slice(1);
diff --git a/dist/pluginWebpack5.js b/dist/pluginWebpack5.js
index af21c6c626f963bb119a6ae62f8e4d5e8ccc5fbf..7049ee535f007fbcb14cb681a03948c9d2b15681 100644
--- a/dist/pluginWebpack5.js
+++ b/dist/pluginWebpack5.js
@@ -40,16 +40,16 @@ const ruleSetCompiler = new RuleSetCompiler([
 ]);
 class VueLoaderPlugin {
     apply(compiler) {
-        // @ts-ignore
-        const normalModule = compiler.webpack.NormalModule || NormalModule;
-        // add NS marker so that the loader can detect and report missing plugin
-        compiler.hooks.compilation.tap(id, (compilation) => {
-            normalModule
-                .getCompilationHooks(compilation)
-                .loader.tap(id, (loaderContext) => {
-                loaderContext[NS] = true;
-            });
-        });
+        // // @ts-ignore
+        // const normalModule = compiler.webpack.NormalModule || NormalModule;
+        // // add NS marker so that the loader can detect and report missing plugin
+        // compiler.hooks.compilation.tap(id, (compilation) => {
+        //     normalModule
+        //         .getCompilationHooks(compilation)
+        //         .loader.tap(id, (loaderContext) => {
+        //         loaderContext[NS] = true;
+        //     });
+        // });
         const rules = compiler.options.module.rules;
         let rawVueRule;
         let vueRules = [];